screengrab/lib/screengrab/reports_generator.rb

```diff
@@ -0,0 +1,36 @@
+require 'erb'
+require 'fastimage'
```

Comment:
One more thought - I don't know what this `require 'fastimage'` is for. It appears to work without it, so let's remove :+1: